config/servicemappings/accesscontextmanager.yaml

```diff
@@ -356,3 +356,34 @@ spec:
                 group: resourcemanager.cnrm.cloud.google.com
               targetField: "number"
               valueTemplate: "projects/{{value}}"
+    - name: google_access_context_manager_service_perimeter_resource
+      kind: AccessContextManagerServicePerimeterResource
+      idTemplate: "{{perimeter_name}}/{{resource}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      v1alpha1ToV1beta1: true
+      storageVersion: v1alpha1
+      resourceReferences:
+        - key: resourceRef
+          tfField: resource
+          description: |-
+            A GCP resource that is inside of the service perimeter.
+          gvk:
+            kind: Project
+            version: v1beta1
+            group: resourcemanager.cnrm.cloud.google.com
+          targetField: number
+          valueTemplate: "projects/{{value}}"
+        - key: perimeterNameRef
+          tfField: perimeter_name
+          description: |-
+            Only the `external` field is supported to configure the reference.
```

Comment:
Are we just relying on field description to prevent users from using `name` and `namespace`? Just brainstorming, one option is to use kustomize to modify the CRD schema so that the field only takes `external`. Another option is to make this field not a reference field.

Reply:
I won't worry about that too much because users won't be able to apply their configuration successfully if they don't use external.